src/EnergyPlus/DataHeatBalance.cc

```diff
@@ -1032,6 +1041,17 @@ namespace DataHeatBalance {
 		}
 	}
 
+
+	void
+	SetZoneWindDirAt()
+	{
+		// Using/Aliasing
+		Real64 const fac( DataEnvironment::WindDir );
+		for ( auto & zone : Zone ) {
+			zone.SetWindDirAt( fac );
```

Comment:
The WindSpeed for ZoneData and DataSurface ware set up in the same way. I was hoping to be consistent - Only WindDir is new.